pkg/tracer/trace.go

```diff
@@ -0,0 +1,52 @@
+package tracer
+
+import (
+	"flag"
+
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/exporters/jaeger"
+	"go.opentelemetry.io/otel/sdk/resource"
+	tracesdk "go.opentelemetry.io/otel/sdk/trace"
+	semconv "go.opentelemetry.io/otel/semconv/v1.4.0"
+	"go.opentelemetry.io/otel/trace"
+)
+
+const (
+	defaultTracerName = "github.com/timescale/promscale"
```

Comment:
Is this a convention?

Reply:
Needed a name for the instrumentation library, seen a couple of places using repo URL so did it like that. Honestly, makes little difference.